db-ops/create-db-user.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# docker image with postgres client only
+DOCKER_IMAGE_TAG=governmentpaas/psql:latest
+
+DB_ENDPOINT=$(aws rds describe-db-instances --region=$REGION --query "DBInstances[?DBName=='$PROJECT_NAME'].Endpoint.Address" | jq -r '.[0]')
+SECRET_ID=$(aws secretsmanager list-secrets --region $REGION  --query "SecretList[?Name=='$PROJECT_NAME-$ENVIRONMENT-rds-$SEED'].Name" | jq -r ".[0]")
+# RDS MASTER
+MASTER_RDS_USERNAME=master_user
+SECRET_PASSWORD=$(aws secretsmanager get-secret-value --region=$REGION --secret-id=$SECRET_ID | jq -r ".SecretString")
+# APPLICATION DB ADMIN
+DB_APP_USERNAME=$PROJECT_NAME
+DB_APP_PASSWORD=$(openssl rand -base64 16)
```

Comment:
Also, sorry, this might be a better way to generate a random string that doesn't require `openssl`:
`$(LC_ALL=C tr -dc 'A-Za-z0-9' < /dev/urandom | base64 | head -c 16)`